Fails to build `base-test` Docker container

**What happened**:
Currently, the build process for the `base-test` Docker container fails with:
```
ln: '/usr/bin/pytest-3' and '/usr/bin/pytest' are the same file
[2105](https://github.com/gardenlinux/gardenlinux/runs/6649592019?check_suite_focus=true#step:4:2106)
Error: error building at STEP "RUN echo "deb http://deb.debian.org/debian testing main contrib non-free" > /etc/apt/sources.list      && apt-get update      && apt-get install -y --no-install-recommends           curl           unzip           ca-certificates           less           apt-transport-https           gnupg           python3-pip           python3-pytest           vim           procps           openssh-client           inetutils-ping           wget           iproute2           libguestfs-tools           qemu-system-x86           qemu-system-arm           shellcheck           build-essential           libpython3-dev      && (test -x /usr/bin/pytest-3 && ln -sf /usr/bin/pytest-3 /usr/bin/pytest)": error while running runtime: exit status 1
[2106](https://github.com/gardenlinux/gardenlinux/runs/6649592019?check_suite_focus=true#step:4:2107)
make[1]: *** [Makefile:37: build-base-test] Error 125
```

--- a/container/base-test/Dockerfile
+++ b/container/base-test/Dockerfile
@@ -26,8 +26,7 @@ RUN echo "deb http://deb.debian.org/debian testing main contrib non-free" > /etc
           qemu-system-arm \
           shellcheck \
           build-essential \
-          libpython3-dev \
-     && (test -x /usr/bin/pytest-3 && ln -sf /usr/bin/pytest-3 /usr/bin/pytest)
+          libpython3-dev
 
 # pipenv package was removed from bookworm https://tracker.debian.org/pkg/pipenv
 RUN pip install pipenv==2022.4.8

